formula/src/main/java/com/instacart/formula/internal/FormulaManagerImpl.kt

```diff
@@ -196,4 +195,19 @@ internal class FormulaManagerImpl<Input, State : Any, Output>(
             key = formula.key(input)
         )
     }
+
+    private fun getOrInitChildTransitionListener(): TransitionListener {
+        return childTransitionListener ?: run {
+            val listener = TransitionListener { transition, isValid ->
+                val frame = this.frame
+                if (!isValid) {
+                    frame?.childInvalidated()
+                }
+                val isValid = frame != null && frame.isValid()
```

Comment:
Looks like this val is shadowing the parent `isValid`. Not a big deal, but might be nice to change.

Reply:
I thought I had fixed it... Renaming passed `isValid` to `isChildValid`